deepset_cloud_sdk/cli.py

```diff
@@ -80,6 +90,80 @@ def list_files(
                 break
 
 
+@cli_app.command()
+def list_upload_sessions(
+    api_key: Optional[str] = None,
+    api_url: Optional[str] = None,
+    is_expired: Optional[bool] = False,
+    workspace_name: str = DEFAULT_WORKSPACE_NAME,
+    batch_size: int = 10,
+    timeout_s: int = 300,
+) -> None:
+    """List files in deepset Cloud.
+
+    A CLI method to list files that exist in deepset Cloud.
+
+    :param api_key: deepset Cloud API key to use for authentication.
+    :param api_url: API URL to use for authentication.
+    :param workspace_name: Name of the workspace to list the files from.
+    :param name: Name of the file to odata_filter for.
+    :param content: Content of the file to odata_filter for.
+    :param odata_filter: odata_filter to apply to the file list.
+    :param batch_size: Batch size to use for the file list.
```

Comment:
need to remove odata_filter, content, name from params
need to add timeout_s to params

Reply:
done